packages/contracts/src/2.0.0/tokens/ERC721Token/MintableERC721Token.sol

```diff
@@ -0,0 +1,83 @@
+/*
+
+  Copyright 2018 ZeroEx Intl.
+
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+
+*/
+
+pragma solidity 0.4.24;
+
+import "./ERC721Token.sol";
+
+
+contract MintableERC721Token is
+    ERC721Token
+{
+
+    /// @dev Function to mint a new token
+    ///      Reverts if the given token ID already exists
+    /// @param _to Address of the beneficiary that will own the minted token
+    /// @param _tokenId ID of the token to be minted by the msg.sender    
+    function _mint(address _to, uint256 _tokenId)
```

Comment:
can we expose a public mintable function for testnets?

Reply:
We have `DummyERC721Token`. Maybe I should just drop the `onlyOwner` modifiers?

Reply:
Going to remove `onlyOwner` from `mint` but leave it on `burn`.